src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_FlowAnalysis.cs

```diff
@@ -958,6 +961,9 @@ static void F(object x, object? y)
                 // (5,10): error CS0019: Operator '??' cannot be applied to operands of type '<null>' and '<null>'
                 //         (null ?? null).ToString();
                 Diagnostic(ErrorCode.ERR_BadBinaryOps, "null ?? null").WithArguments("??", "<null>", "<null>").WithLocation(5, 10),
+                // (5,10): warning CS8602: Possible dereference of a null reference.
+                //         (null ?? null).ToString();
+                Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "null ?? null").WithLocation(5, 10),
```

Comment:
>WRN_NullReferenceReceiver [](start = 37, length = 25)

Nit: Seems like a cascading diagnostic that we could avoid.

Reply:
Even though the expression is an error, the top-level nullability is clear.

---
In reply to: [189332910](https://github.com/dotnet/roslyn/pull/26356#discussion_r189332910) [](ancestors = 189332910)